src/core/EventBus.js

```diff
@@ -49,11 +51,13 @@ function EventBus() {
 
         var handler = {
             callback: listener,
-            scope: scope
+            scope: scope,
+            priority: priority !== undefined ? priority : LOW_PRIORITY
```

Comment:
We can get rid of this kind of construct in the code by using [default parameters](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Default_parameters), which Babel will transpile into basically the same thing.